js/main.js

```diff
@@ -365,6 +365,13 @@
       parseRepos(ghApi, [$(this).data().repoPath]);
     });
 
+    $('#approved-prs').on('click', 'td', function() {
+      var link = $(this).find('a');
+      if (link.length === 1 && link.attr('href')) {
+        window.open(link.attr('href'));
```

Comment:
You should probably either use `link.click();` or add a second parameter to [window.open](https://developer.mozilla.org/en-US/docs/Web/API/Window/open).  These links typically open in a new tab, but here I don't think they are.